infrastructure/pom.xml

```diff
@@ -61,5 +61,34 @@
             <groupId>org.wso2.carbon.testgrid</groupId>
             <artifactId>org.wso2.carbon.testgrid.utils</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.amazonaws</groupId>
+            <artifactId>aws-java-sdk</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
Shall we add a comment stating that this dependency is required by AWS?

Reply:
Done